.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
```

Comment:
This can be replaced by https://github.com/puppetlabs/pdk-templates/blob/main/moduleroot/.github/workflows/spec.yml.erb (but you might need to track `main` for future updates.

Reply:
That looks way more complicated that the simple action I've added here... any particular reasons I might want to use that one instead?